### ISSUE TITLE
Contrast issue in pages

### ISSUE THREAD

**Issue by ManishBisht777** [OWASP/BLT#1013]
![image](https://user-images.githubusercontent.com/89926834/210370050-3f676e3e-e219-4fbb-b961-ce2a86bd3fcd.png)

The contrast ratio of gray color text `is 2.85` which Does not meet WCAG standards for accessible text 

Recommended (AA rating)

![image](https://user-images.githubusercontent.com/89926834/210370535-da46c08d-0c4c-4549-9fb8-286ba14d57ad.png)

**Comment by ManishBisht777** [comment]
> Good find. Let’s go for an AAA rating.

isn't that for government sites only?

**Comment by fredfalcon** [comment]
I don’t know, but we could use government designs - they are free https://designsystem.digital.gov/

**Comment by co-decode** [comment]
I have fixed the contrast of the text on the index page.
If you find some more cases of low contrast in the website, feel free to contribute!
